src/controllers/trust.information.controller.ts

```diff
@@ -38,6 +39,19 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
       trusts: trustData
     };
 
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    const trustErrors = checkMandatoryTrustFields(trustData);
+    if ( trustErrors.length > 0 ) {
+      return res.render(config.TRUST_INFO_PAGE, {
```

Comment:
it looks like appData is only needed by the res.render so I would move
const appData: ApplicationData = getApplicationData(req.session);
inside the 
if ( trustErrors.length > 0 ) {

so that we only make the call if it is needed